docs/source/notes/randomness.rst

```diff
@@ -121,3 +123,38 @@ CUDA RNN and LSTM
 In some versions of CUDA, RNNs and LSTM networks may have non-deterministic behavior.
 See :meth:`torch.nn.RNN` and :meth:`torch.nn.LSTM` for details and workarounds.
 
+CUDA random seed
+----------------
+
+Also it is recommended to set CUDAs random seed::
+
+    torch.cuda.manual_seed_all(0)
+    torch.cuda.manual_seed(0)
+
+Python
```

Comment:
This should be inside "controlling sources of randomness" above no?

Reply:
Agree, moved